tests/test_gradient_clipping.py

```diff
@@ -0,0 +1,144 @@
+from unittest.mock import MagicMock
+
+import torch
+
+from modalities.training.gradient_clipping.fsdp_gradient_clipper import (
+    DummyGradientClipper,
+    FSDP1GradientClipper,
+    FSDP1LoggingOnlyGradientClipper,
+    FSDP2GradientClipper,
+    FSDP2LoggingOnlyGradientClipper,
+    GradientClippingMode,
+)
+
+
+class MockFSDP2Model:
+    def __init__(self):
+        self.param1 = torch.nn.Parameter(torch.tensor([1.0, 2.0]))
+        self.param2 = torch.nn.Parameter(torch.tensor([3.0, 4.0]))
+        self.param1.grad = torch.tensor([1.0, 1.0])
+        self.param2.grad = torch.tensor([1.0, 1.0])
+
+    def parameters(self):
+        return [self.param1, self.param2]
+
+
+# Note: Replace 'your_module' above with the correct module path where the gradient clipping classes are defined.
+
+
+def test_fsdp1_gradient_clipper():
+    """
+    Test that FSDP1GradientClipper correctly calls the wrapped model's clip_grad_norm_ method
+    with the specified max_norm and norm_type.
+    """
+    # Create a mock FSDP1 model
+    mock_model = MagicMock()
+    max_norm = 1.0
+    norm_type = GradientClippingMode.P2_NORM
+    clipper = FSDP1GradientClipper(wrapped_model=mock_model, max_norm=max_norm, norm_type=norm_type)
+
+    # Call clip_gradients
+    clipper.clip_gradients()
+
+    # Verify that clip_grad_norm_ was called with the correct arguments
+    mock_model.clip_grad_norm_.assert_called_once_with(max_norm=max_norm, norm_type=norm_type.value)
+    # Note: The actual norm returned depends on the mock's return value, which isn't tested here
```

Comment:
Can we also check here if the gradients are scaled down to the expected values, like we did for the FSDP2 tests?